plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OracleConfig.java

```diff
@@ -141,6 +142,19 @@ public OracleConfig setInactiveConnectionTimeout(Duration inactiveConnectionTime
         return this;
     }
 
+    public Optional<@Min(0) Integer> getJdbcFetchSize()
+    {
+        return Optional.ofNullable(jdbcFetchSize);
+    }
+
```

Comment:
Do we want to have corresponding session properties for this configuration?

Reply:
To be honest not sure, I thought about this property in the way, that generally it should not be touched at least for users, who really knows what they are doing.